nationality-park/style.scss

```diff
@@ -0,0 +1,160 @@
+/*MAIN SETTINGS*/
+$backgroundColor: rgb(30, 30, 30);
+$backgroundColor50: rgba(30, 30, 30, 0.5);
+$accentColor: green;
+$textColor: whitesmoke;
+*{
```

Comment:
1. make a space between selector and opening curly bracket
2. make single line between each block of styling e.g.:

.special {
  color: red;
}

p {
  color: blue;
}